cmd/geth/chaincmd.go

```diff
@@ -278,7 +278,8 @@ func importChain(ctx *cli.Context) error {
 	utils.SetupMetrics(ctx)
 	// Start system runtime metrics collection
 	go metrics.CollectProcessMetrics(3 * time.Second)
-	stack, _ := makeFullNode(ctx)
+	// Make the bare-minimum node to avoid unnecessary service start-up
```

Comment:
Would be better to put a documentation comment on the actual function definition instead of documenting the behavior of makeConfigNode at individual call sites.

```
makeConfigNode loads geth configuration and creates a blank node instance.
```

```
makeFullNode loads geth configuration and creates the Ethereum backend.
```